### ISSUE TITLE
some client example not working

### ISSUE THREAD

**Issue by 435352980** [flowtsohg/mdx-m3-viewer#34]
Does it change some APIs now？
The `War3MapViewer` seemed to have default  scene in the 'scenes' prop, should replace `viewer.scene` by `viewer.scenes[0]`?

![image](https://user-images.githubusercontent.com/35485428/75783639-d69aba80-5d9b-11ea-8c16-bf76801b49d6.png)

https://github.com/flowtsohg/mdx-m3-viewer/blob/ac7e6c2c5d4698d2180533824d1d9785d6ad4b63/clients/map/index.js#L80-L91

And in some other examples, seemed model viewer  not imported correctly.
https://github.com/flowtsohg/mdx-m3-viewer/blob/ac7e6c2c5d4698d2180533824d1d9785d6ad4b63/clients/example/index.js#L14
It should be 
```js
let viewer = new ModelViewer.default.viewer.ModelViewer(canvas);
```
https://github.com/flowtsohg/mdx-m3-viewer/blob/ac7e6c2c5d4698d2180533824d1d9785d6ad4b63/clients/recorder/index.js#L4
It should be 
```js
let quat = common.glMatrix.quat;
```
https://github.com/flowtsohg/mdx-m3-viewer/blob/ac7e6c2c5d4698d2180533824d1d9785d6ad4b63/clients/sanitytest/index.js#L3-L5
It should be 
```js
let glMatrix = ModelViewer.common.glMatrix;
let vec3 = glMatrix.vec3;
let quat = glMatrix.quat;
```

The class `TriggerData` seems to have not been exported
https://github.com/flowtsohg/mdx-m3-viewer/blob/ac7e6c2c5d4698d2180533824d1d9785d6ad4b63/clients/weu/main.js#L10-L11

**Comment by flowtsohg** [comment]
`War3MapViewer.scene` was renamed to `War3MapViewer.worldScene`. Either way they are equivalent to `War3MapViewer.scenes[0]`.
I did move around exports and imports in the previous two commits or so. It will be nice if you can submit a PR :P

**Comment by flowtsohg** [comment]
Fixed by #35 thanks
